symengine/integer.cpp

```diff
@@ -78,9 +78,13 @@ RCP<const Number> Integer::pow_negint(const Integer &other) const
 {
     RCP<const Number> tmp = powint(*other.neg());
     if (is_a<Integer>(*tmp)) {
-        rational_class q(mp_sign(static_cast<const Integer &>(*tmp).i),
-                         mp_abs(static_cast<const Integer &>(*tmp).i));
-        return make_rcp<const Rational>(std::move(q));
```

Comment:
Thanks for catching this. what should be done to fix this is to use, `Rational::from_mpq` instead of `make_rcp`